### PR TITLE
bring Solution to build on Mac OS X with mono & FAKE

### PR DESCRIPTION
**Pull request by SabotageAndi**
Following is done:
.) added scripts for cmd & bash build
.) downgrade Numsense.UnitTests.CSharp.csproj to .Net v4.5 - v4.6 is not yet supported in mono/xbuild
.) fixed paths in nuspec
.) add root files to Solution

### CODE REVIEW

File: Numsense.sln [hunk 1]
```diff
@@ -9,6 +9,16 @@ Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Numsense.UnitTests", "Numse
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Numsense.UnitTests.CSharp", "Numsense.UnitTests.CSharp\Numsense.UnitTests.CSharp.csproj", "{5FEBD8A6-3ED8-4E4F-B484-DE4CB77B6034}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{98273287-AB32-4958-B4C2-47D77A7D42F0}"
+	ProjectSection(SolutionItems) = preProject
```
Comment (ploeh) [inline Numsense.sln]: What's the benefit of adding these files to the Visual Studio solution?

**Comment by ploeh** [comment]
Thank you for your interest in contributing to Numsense! It's looking good :+1: 

It runs and passes all tests on my machine, but I've only been able to test that it (still) works on Windows. I presume that you've tested that it works on Mac OS X :smile:

File: Numsense.sln (hunk 1 above)
Comment (SabotageAndi) [inline Numsense.sln]: When I am working with Visual Studio I like to have all files that are edited in the solution explorer.

**Comment by SabotageAndi** [comment]
Yes, I have done the most stuff for it on Mac OS X.
I also checked it on Windows with VS 2015.

File: Numsense.sln (hunk 1 above)
Comment (ploeh) [inline Numsense.sln]: It's not my cup of tea, but what the heck! Let's do this :smile:

**Comment by ploeh** [comment]
Thank you for your contribution :+1:

**Pull request merged**